converter/src/main/java/org/geysermc/pack/converter/util/VanillaPackProvider.java

```diff
@@ -36,11 +37,13 @@
 import java.net.URL;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.stream.Stream;
 
 public final class VanillaPackProvider {
     private static final String JAR_DOWNLOAD = "https://piston-data.mojang.com/v1/objects/%s/client.jar";
-    private static final String JAR_HASH = "0c3ec587af28e5a785c0b4a7b8a30f9a8f78f838";
+    private static final String JAR_HASH = "a2db1ea98c37b2d00c83f6867fb8bb581a593e07";
```

Comment:
Might be worth it to reference the Geyser code for downloading the java client jar - then this would not need to be hardcoded :)